.github/CODEOWNERS

```diff
@@ -97,12 +97,18 @@ assets/               @DataDog/agent-integrations
 
 
 # Database monitoring
-**/postgres/statements.py                    @DataDog/agent-integrations @DataDog/database-monitoring
-**/postgres/statement_samples.py             @DataDog/agent-integrations @DataDog/database-monitoring
-**/postgres/tests/test_statements.py         @DataDog/agent-integrations @DataDog/database-monitoring
-**/mysql/statements.py                       @DataDog/agent-integrations @DataDog/database-monitoring
-**/mysql/statement_samples.py                @DataDog/agent-integrations @DataDog/database-monitoring
-**/mysql/tests/test_statements.py            @DataDog/agent-integrations @DataDog/database-monitoring
+/base/utils/db/utils.py                          @DataDog/agent-integrations @DataDog/database-monitoring
+datadog_checks_base/tests/test_db_util.py        @DataDog/agent-integrations @DataDog/database-monitoring
```

Comment:
Same here. I don't think this will work as it's similar to the last pattern that we had to fix by adding a wildcard prefix https://github.com/DataDog/integrations-core/pull/9565

Reply:
In this case it's different because this is the full path, while in the case of utils the full path is: `datadog_checks_base/datadog_checks/base/utils/db/utils.py`